lib/completion.ts

```diff
@@ -92,8 +102,11 @@ export class Completion implements CompletionInstance {
       this.usage.getCommands().forEach(usageCommand => {
         const commandName = parseCommand(usageCommand[0]).cmd;
         if (args.indexOf(commandName) === -1) {
-          if (!this.zshShell) {
+          if (this.bashShell) {
```

Comment:
It is hard to reason about what shell gets handled where here. The original code had basically zsh and not zsh. I suggest keep the same sort of logic, so it is a more obvious diff, and easier to understand what gets handled where.

```
if (this.fishShell) {
} else if (this.zshShell) {
} else {
// bash et al
}